lab02/index.html

```diff
@@ -0,0 +1,354 @@
+                        <h3 class="home_profession">Computer Science student</h3>
+                    </div>
+
+                    <!-- WEBSITE SECTION -->
+
+                    <section class="section_website">
+                        <hr>
+                        <p>This CV was coded! <br/>
+                            For more info, check out <br/>my website.</p>
+                        <div class="website_link">
+                            <a href="https://linamoussadek.github.io/MyOwnWebsite/" target="_blank">linamoussadek.github.io/MyOwnWebsite/</a>
+                        </div>
+                        <hr>
+                        <!-- Button to generate and download the pdf. Available for desktop. -->
+                        <div>
+                            <a download="" href="assets/pdf/Lina-Moussadek-CV.pdf"
+                               class="home_button-movil">Download</a>
+                        </div>
+                    </section>
+
+                    <div class="home_address bd-grid">
+                                <span class="home_information">
+                                    <i class="bx bx-map"> &nbsp; </i> Ottawa, Ontario
+                                </span>
+                        <span class="home_information">
+                                    <i class="bx bx-envelope"> &nbsp; </i> lmous090@uottawa.ca
+                                </span>
+                        <div class="social_container bd-grid">
+                            <a href="https://www.linkedin.com/in/lina-moussadek/" target="_blank" class="social_link">
+                                <i class="bx bxl-linkedin-square social_icon"></i> Lina Moussadek
+                            </a>
+                            <a href="https://github.com/linamoussadek/" target="_blank" class="social_link">
+                                <i class="bx bxl-github social_icon"></i> @linamoussadek
+                            </a>
+                        </div>
+                    </div>
+                </div>
+
+                <!-- Theme change button -->
+                <i class="bx bx-moon change-theme" title="Theme" id="theme-button"></i>
+                <!-- Button to generate and download the pdf. Available for desktop -->
+                <i class="bx bx-download generate-pdf" title="Generate PDF" id="resume-button"></i>
+            </section>
+
+            <!-- PROFILE -->
+            <section class="profile section" id="profile">
+                <h2 class="section-title">About</h2>
+                <p class="profile_description">
+                    I was born in Casablanca, Morocco where I attended a French school.
+                    I am looking for a challenging internship for Fall 2024.
+                    I have a Canadian Work Permit and am happy to relocate anywhere!
+                </p>
+            </section>
+
+            <!-- SKILLS  -->
+            <section class="skills section" id="#skills">
```

Comment:
No need for a pound key, you can simply write it as id="skills"

Reply:
Addressed